features/admin.policy-administration.v1/components/policy-list-node.tsx

```diff
@@ -140,17 +134,17 @@ const PolicyListNode: FunctionComponent<PolicyListDraggableNodePropsInterface> =
                                 <Icon
                                     onClick={ handleActivate }
                                     data-componentid={ `${componentId}-edit-button` }
-                                    className="list-icon"
+                                    className="policy-list-icon list-icon"
                                     size="massive"
                                     color="grey"
                                     name="chevron left"
                                 />
                             ) }
                             position="top center"
-                            content={ "Deactivate" }
+                            content={ "Activate" }
```

Comment:
Let's use i18n for this string literal.

```suggestion
                            content={ "t("common:activate")" }
```